.coveragerc

```diff
@@ -1397,6 +1397,9 @@ omit =
     homeassistant/components/w800rf32/*
     homeassistant/components/waqi/sensor.py
     homeassistant/components/waterfurnace/*
+    homeassistant/components/waterkotte_heatpump/__init__.py
+    homeassistant/components/waterkotte_heatpump/entity.py
+    homeassistant/components/waterkotte_heatpump/sensor.py
```

Comment:
You should probably create test for these. But I don't think it is strictly required.